helper/resource/testing.go

```diff
@@ -20,6 +21,68 @@ import (
 	"github.com/hashicorp/terraform/terraform"
 )
 
+// FlagSweep is a flag available when running tests on the command line. This
+// flag bypasses the normal Test path and instead runs functions designed to
+// clean up any leaked resources a testing environment could have created. It is
+// a best effort attempt, and relies on Provider authors to implement "Sweeper"
+// methods for resources. Adding Sweeper methods with AddTestSweepers will
+// construct a list of sweeper funcs to be called here. We iterate through the
+// tests, and exit on any errors. At time of writing, sweepers are ran
+// sequentially.
+//
+// WARNING:
+// Sweepers are designed to be destructive. You should not use the -sweep flag
+// in any environment that is not strictly a test environment. Resources will be
+// destroyed.
+
+var FlagSweep bool
```

Comment:
This probably can be unexported.

Also, you can make this a one-liner with `var flagSweep = flag.Bool(...)` and access it with `*flagSweep`